Does the job has information on which compute node it landed? We've checked our cluster and that particular file is confirmed to be accessible on all live nodes.



-----Original Message-----
From: helpdesk@ggus.org [mailto:helpdesk@ggus.org] 
Sent: Thursday, October 15, 2015 10:08 AM
To: t2-admins@phys.ufl.edu; bockjoo@phys.ufl.edu
Subject: GGUS-Ticket-ID: #116943 Ticket for site "UFlorida-HPC" "cms" "T2_US_Florida suffers from low job efficiency in production"


Dear support staff,
ticket #116943 for site "UFlorida-HPC" is ASSIGNED to you. 

REFERENCE LINK: https://ggus.eu/index.php?mode=ticket_info&ticket_id=116943
SUBJECT: T2_US_Florida suffers from low job efficiency in production

-----------------------------------------------------------------------------
TICKET INFORMATION:
-----------------------------------------------------------------------------
DESCRIPTION ->
Lots of jobs with exit code 60515, exemplary log below.

++++++++++

    Job Name: 4098e96c-72f9-11e5-9074-a0369f23d008-170
    WMBS job id: 1605648
    Workflow: pdmvserv_task_SMP-RunIIWinter15wmLHE-00048__v1_T_151013_164323_7138
    Task: /pdmvserv_task_SMP-RunIIWinter15wmLHE-00048__v1_T_151013_164323_7138/SMP-RunIIWinter15wmLHE-00048_0
    Status: jobcooloff
    Input dataset:
    Site: T2_US_Florida
    Agent: cmssrv218.fnal.gov
    ACDC URL: https://cmsweb.cern.ch/couchdb/acdcserver
    State Transition
        jobcooloff: 2015/10/15 (Thu) 05:15:50 UTC, T2_US_Florida
    Exit code: 60515
    Retry count: 0
    Errors:
        logArch1
        cmsRun1
            SCRAMScriptFailure (Exit Code: 60515)

                SCRAM scripts failed to run!
                PreScriptScramFailure
                Message: Error running command
                /cvmfs/cms.cern.ch/COMP/slc6_amd64_gcc481/external/python/2.6.8-comp9/bin/python2.6 -m WMCore.WMRuntime.ScriptInvoke WMTaskSpace.cmsRun1 SetupCMSSWPset 

                5
                 
                        Scram Command Diagnostic:
                        Command : scramv1
                        Architecture: slc6_amd64_gcc481
                        Executed:
/cvmfs/cms.cern.ch/slc6_amd64_gcc481/cms/cmssw/CMSSW_7_1_19/external/slc6_amd64_gcc481/bin/python2.7 -m WMCore.WMRuntime.ScriptInvoke WMTaskSpace.cmsRun1 SetupCMSSWPset 

                        Exit Status: 5
                        Stdout: None
                        Stderr: None 
                	ModuleName : WMCore.WMSpec.Steps.WMExecutionFailure
                	MethodName : __init__
                	ClassInstance : None
                	FileName :
/local/scratch/16332432.moab.ufhpc/glide_oOakdY/execute/dir_4883/job/WMCore.zip/WMCore/WMSpec/Steps/WMExecutionFailure.py
                	ClassName : None
                	LineNumber : 18
                	ErrorNr : 60515

                Traceback: 


                 Adding last ten lines of SCRAM error log:
                Log for recording SCRAM command-line output
                -------------------------------------------
                Traceback (most recent call last):
                  File "/cvmfs/cms.cern.ch/slc6_amd64_gcc481/external/python/2.7.6/lib/python2.7/runpy.py", line 162, in _run_module_as_main
                    "__main__", fname, loader, pkg_name)
                  File "/cvmfs/cms.cern.ch/slc6_amd64_gcc481/external/python/2.7.6/lib/python2.7/runpy.py", line 72, in _run_code
                    exec code in run_globals
                  File
"/local/scratch/16332432.moab.ufhpc/glide_oOakdY/execute/dir_4883/job/WMCore.zip/WMCore/WMRuntime/ScriptInvoke.py",
line 19, in 
                  File
"/local/scratch/16332432.moab.ufhpc/glide_oOakdY/execute/dir_4883/job/WMCore.zip/WMCore/WMRuntime/ScriptFactory.py",
line 9, in 
                  File
"/local/scratch/16332432.moab.ufhpc/glide_oOakdY/execute/dir_4883/job/WMCore.zip/WMCore/WMFactory.py", line 13, in 
                  File
"/cvmfs/cms.cern.ch/slc6_amd64_gcc481/external/python/2.7.6/lib/python2.7/logging/__init__.py", line 26, in 
                    import sys, os, time, cStringIO, traceback, warnings, weakref
                ImportError: dynamic module does not define init function (inittraceback)

        stageOut1
            ReportManipulatingError (Exit Code: 99999)

                Could not find report file for step stageOut1!

    Input files:
    Lumis:
    Output:
        logArchive
            lfn:
/store/unmerged/logs/prod/2015/10/15/pdmvserv_task_SMP-RunIIWinter15wmLHE-00048__v1_T_151013_164323_7138/SMP-RunIIWinter15wmLHE-00048_0/0004/0/4098e96c-72f9-11e5-9074-a0369f23d008-170-0-logArchive.tar.gz
            location: T2_US_Florida
            pfn:
            checksums: adler32: 47d5c020
            size: 0
    log archive:

LAST MODIFIER -> alex.van.spilbeeck
NOTIFIED SITE -> UFlorida-HPC
CONCERNED VO -> cms
PRIORITY -> urgent
TYPE OF ISSUE -> CMS_Central Workflows
SUBMITTER -> alex.van.spilbeeck

*********************************************************************************
 This is an automated mail. When replying don't change the subject line!
 Type your text above this box and S T R I P  P R E V I O U S  M A I L S please!!
*********************************************************************************


